notebooks/getting_started.ipynb

```diff
@@ -127,18 +173,74 @@
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "100%|██████████| 3/3 [00:25<00:00,  8.46s/it]"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "LF 0: Accuracy 57%, Abstain rate 56%\n",
+      "LF 1: Accuracy 91%, Abstain rate 93%\n",
+      "LF 2: Accuracy 58%, Abstain rate 97%\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "\n"
+     ]
+    }
+   ],
    "source": [
     "from typing import List\n",
     "from deep_patient_cohorts import POSITIVE, NEGATIVE, ABSTAIN\n",
     "\n",
+    "# heart disease LF\n",
     "def heart_disease(self, texts: List[str]) -> List[int]:\n",
     "    return [POSITIVE if \"heart disease\" in text.text.lower() else ABSTAIN for text in texts]\n",
     "\n",
     "labeler.add(heart_disease)\n",
     "\n",
+    "# st elevation LF\n",
+    "def st_elevation(self, texts: List[str]) -> List[int]:\n",
+    "    search_list = [\"STEMI\", \"ST elevation\", \"ST elevation MI\"]\n",
+    "    return [POSITIVE if any([x in text.text.lower() for x in search_list]) else ABSTAIN for text in texts]\n",
+    "\n",
+    "# st elevation LF\n",
+    "def st_elevation(self, texts: List[str]) -> List[int]:\n",
+    "    search_list = [\"STEMI\", \"ST elevation\", \"ST elevation MI\"]\n",
+    "    return [POSITIVE if any([x in text.text.lower() for x in search_list]) else ABSTAIN for text in texts]\n",
+    "\n",
+    "# atherosclerosis\n",
+    "def atherosclerosis(self, texts: List[str]) -> List[int]:\n",
+    "    search_list = [\"atherosclerosis\", \"arteriosclerosis\", \"atherosclerotic\", \"arterial sclerosis\", \"artherosclerosis\", \"atherosclerotic disease\"]\n",
+    "    return [POSITIVE if any([x in text.text.lower() for x in search_list]) else ABSTAIN for text in texts]\n",
+    "\n",
+    "# heart_attack\n",
+    "def heart_attack(self, texts: List[str]) -> List[int]:\n",
+    "    search_list = [\"myocardial infarcation\", \"MI\", \"ischemic heart disease\", \"cardiac arrest\", \"coronary infarction\", \"asystole\", \"cardiopulmonary arrest\", \"coronary thrombosis\", \"heart arrest\", \"heart attack\", \"heart stoppage\"]\n",
+    "    return [POSITIVE if any([x in text.text.lower() for x in search_list]) else ABSTAIN for text in texts]\n",
+    "\n",
+    "# heart_failure\n",
+    "def heart_failure(self, texts: List[str]) -> List[int]:\n",
+    "    search_list = [\"congestive heart failure\", \"decomensated heart failure\", \"CHF\", \"left-side heart failure\", \"right-sided heart failure\"]\n",
+    "    return [POSITIVE if any([x in text.text.lower() for x in search_list]) else ABSTAIN for text in texts]\n",
+    "\n",
+    "labeler.add(heart_disease)\n",
+    "labeler.add(st_elevation)\n",
+    "labeler.add(atherosclerosis)\n",
+    "labeler.add(heart_attack)\n",
+    "labeler.add(heart_failure)\n",
+    "\n",
```

Comment:
Did you get a chance to check the accuracy of any of these? If not no worries, I will do it. We should probably start to include only the acceptably performing LFs in the repo so it doesnt' just become a dump.